docs/claimantmodel/Maps.md

```diff
@@ -0,0 +1,115 @@
+# Claimant Model: Maps
+
+This doc presents a model for transparent ecosystems built around maps. This doc requires understanding the core [Claimant Model](CoreModel.md), and it is recommended that [Claimant Model: Logs](Logs.md) is also read first.
+
+## Groundwork: Discoverability
+
+A Claim is discoverable if it can be found by the Claim Verifier without information being passed to them by the Believer.
+The Log Claimant Model states that the purpose of Log Transparency is to enable discoverability of Claims.
+To understand whether this statement applies to Maps, we need to understand what it is about Logs that enables discoverability.
+
+A Log can be thought of as a Map where the key is the leaf index, and new entries need to be written into the next available leaf.
```

Comment:
I think it's worth introducing the key concept of a key (:D) somewhere above this line - the idea of a key in a log is a bit strange without context, but maybe if we just casually use the phrase "key value map" somewhere (maybe in the first sentence?) I think it'll cue this up nicely.